pages/common/bindkey.md

```diff
@@ -0,0 +1,20 @@
+# bindkey
+
+> Add keybindings to Z-Shell.
+> More information: <https://zsh.sourceforge.io/Guide/zshguide04.html>.
+
+- Bind a hotkey to a command:
+
+`bindkey '^k' kill-line`
+
+- Bind a hotkey to arbitary key sequence:
```

Comment:
Where is the sequence in the example?
```suggestion
- Bind a hotkey to a specific key sequence:
```

Reply:
I wanted to say that it can bind a hotkey to a string (like `cd ..\n` in the example). So if user presses Ctrl+O, zsh will act like the user entered `cd ..` then hit enter.

Reply:
Oh, ok.